spec/money_spec.rb

```diff
@@ -181,29 +181,35 @@
     end
 
     it "accepts an optional currency" do
+      Money.default_currency = :usd
       expect(Money.from_amount(1).currency).to eq Money.default_currency
+      Money.default_currency = nil
+
       jpy = Money::Currency.wrap("JPY")
       expect(Money.from_amount(1, jpy).currency).to eq jpy
       expect(Money.from_amount(1, "JPY").currency).to eq jpy
     end
 
     it "accepts an optional bank" do
-      expect(Money.from_amount(1).bank).to eq Money.default_bank
+      expect(Money.from_amount(1, "USD").bank).to eq Money.default_bank
       bank = double "bank"
       expect(Money.from_amount(1, "USD", bank).bank).to eq bank
     end
 
     it 'rounds using rounding_mode' do
-      expect(Money.from_amount(1.999).to_d).to eq 2
+      expect(Money.from_amount(1.999, "USD").to_d).to eq 2
       expect(Money.rounding_mode(BigDecimal::ROUND_DOWN) do
-        Money.from_amount(1.999).to_d
+        Money.from_amount(1.999, "USD").to_d
       end).to eq 1.99
     end
 
     context 'given a currency is provided' do
       context 'and the currency is nil' do
         let(:currency) { nil }
 
+        before { Money.default_currency = :gbp }
+        after { Money.default_currency = nil }
```

Comment:
Same here, you want there to be just one place in the code where a default value is set, cause having it in multiple places can potentially come out of sync